docs/rules/no-html-safe.md

```diff
@@ -0,0 +1,112 @@
+# no-html-safe
+
+`htmlSafe` marks a string as safe for unescaped output with Ember templates so you can render it as HTML. While useful this can inadvertently open you up to Cross-site Scripting (XSS) vulnerabilities, especially if the string was generated from user input or some other untrusted source. **You should only ever use `htmlSafe` with trusted or sanitized input**.
+
+## Rule Details
+
+This rule prevents importing the `htmlSafe` utillity from `@ember/template` (or `@ember/string` for older Ember versions);
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```js
+import { htmlSafe } from '@ember/template';
+```
+
+```js
+import { htmlSafe } from '@ember/string';
+```
+
+## Alternatives
```

Comment:
There are no exact alternatives to using `htmlSafe` so I put together this `Alternatives` section which outlines some ways you can minimise your usage of `htmlSafe`. Happy for feedback whether this is suitable or not.